schema/commands.json

```diff
@@ -31,13 +31,29 @@
       {
         "name": "deploy-app",
         "command": "jhub-apps:deploy-app",
+        "args": {
+          "origin": "toolbar"
+        },
         "label": "Deploy App"
       }
     ]
   },
   "title": "jupyterlab-jhub-apps",
   "description": "jupyterlab-jhub-apps custom command settings.",
   "type": "object",
-  "properties": {},
+  "properties": {
```

Comment:
I moved to `properties` as I was unable to find another way to update the settings values at runtime. This is useful for situations like testing, where I want to test different settings values. I took this approach from https://github.com/jupyterlab/extension-examples/tree/main/settings.

I also noticed that if a user updates their settings, then the application will not notice this. I have now used `properties` and `settings.changed.connect` below to handle this.